src/ripple/basics/Slice.h

```diff
@@ -143,6 +166,11 @@ class Slice
     }
 };
 
+/** An immutable linear range of bytes. */
+using Slice = SliceImpl<false>;
+/** A mutable linear range of bytes. */
+using MutableSlice = SliceImpl<true>;
```

Comment:
Food for thought: the [Networking TS](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/n4734.pdf) defines two very similar types named `mutable_buffer` and `const_buffer`.

Reply:
Thanks. Yeah, we might want to get rid of our slice classes all together and replace them (but obviously, not now).